presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -1647,6 +1648,19 @@ public boolean isHudiMetadataEnabled()
         return this.hudiMetadataEnabled;
     }
 
+    @Config("hive.hudi-tables-use-merged-view")
+    @ConfigDescription("For Hudi tables prefer to fetch the list of files from the merged file system view")
```

Comment:
I think the description should be more clear about the type of value it's looking for. The type is a string, so for users it should be more explicit what the value should be. When I read this I initially thought it should be a boolean.

Based on looking at the tests it looks like it needs a dot-separated name of schema and table? If so, it should be clear about which tables should be added here

```suggestion
    @ConfigDescription("For Hudi tables, A comma-separated list in the form of <schema>.<table> which should prefer to fetch the list of files from the merged file system view")
```

Reply:
Good suggestion, thanks. 
I have updated the config docs at both places